lib/thread.c

```diff
@@ -1046,21 +1046,29 @@ struct thread *funcname_thread_add_event(struct thread_master *m,
  *   - POLLIN
  *   - POLLOUT
  */
-static void thread_cancel_rw(struct thread_master *master, int fd, short state)
+static void thread_cancel_rw(struct thread_master *master, int fd, short state,
+			     int hint)
```

Comment:
name it fd_hint maybe?

Reply:
how about "idx_hint"? it's an index into an array, not an actual fd